Report abort traces to stderr instead of terminal.  This enables apport
to catch the error so it can file a bug report about the crash.

Index: xorg-server/hw/xfree86/common/xf86Init.c
===================================================================
--- xorg-server.orig/hw/xfree86/common/xf86Init.c	2011-08-24 12:56:49.555650622 +0300
+++ xorg-server/hw/xfree86/common/xf86Init.c	2011-08-24 12:56:49.665650623 +0300
@@ -866,6 +866,9 @@
 
   if (!beenHere) {
     umask(022);
+    /* have glibc report internal abort traces to stderr instead of
+       the controlling terminal */
+    setenv("LIBC_FATAL_STDERR_","1",0);
     xf86LogInit();
   }
 
